<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="hlMainSections" stretch="8,1,40">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="vlUISelect" stretch="1,6,1,1,5,1,1,2,0,0,12">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="leInformation">
          <property name="text">
           <string>Parts, tooling, and supplies</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listInformation">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>155</height>
           </size>
          </property>
          <property name="currentRow">
           <number>-1</number>
          </property>
          <item>
           <property name="text">
            <string>search for parts</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>baseplates</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sensors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PCBs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>protomodules</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>modules</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>tooling</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>supplies</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>Production steps and testing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listAssembly">
          <item>
           <property name="text">
            <string>kapton placement steps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sensor placement steps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PCB placement steps</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>wirebonding and encapsulating</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>unbiased DAQ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IV curve</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>Shipping and receiving</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listShippingAndReceiving">
          <item>
           <property name="text">
            <string>shipments</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbUpload">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Upload changes to database (WIP)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlUIpane">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="swPages">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
